### ISSUE TITLE
bug(ios): Short press on the globe key does not switch to the next keyboard

### ISSUE THREAD

**Issue by bharanidharanj** [keymanapp/keyman#11204]
### Describe the bug

I noticed that a short press on the globe key does not switch to the next keyboard.

### Reproduce the bug

1. Install Keyman 17.0.303-beta
2. Download and Install Khmer Angkor, US Basic and Cameroon Qwerty keyboards. 
3. Restart Keyman In-app.
4. Short Press on the globe key to Switch next available keyboard. 

Here, I noticed that it does not switch to the next keyboard. I have attached the Screenshot for reference. 

   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/889212c2-8749-467a-b442-33f06bc0db29">

### Expected behavior

A short press on the globe key should switch the keyboard to another one.

### Keyman apps

- [ ] Keyman for Android
- [X] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.303-beta

### Operating system

iOS 17.4

### Device

iPhone 13 Mobile

**Comment by jahorton** [comment]
Doesn't it just display the keyboard picker when you do this in-app?  The behavior in-app is different than in the system keyboard.

**Comment by bharanidharanj** [comment]
> Doesn't it just display the keyboard picker when you do this in-app? The behavior in-app is different than in the system keyboard.

@jahorton Yes, It displays the keyboard picker menu in the Keyman In-app. However, short press the globe key switches to another keyboard while it is used as system-wide keyboard. So, Is this an expected behavior?

**Comment by jahorton** [comment]
As this matches existing 16.0 behavior for iOS and is more of a feature-request to achieve parity with Keyman for Android, I feel we should defer this until 18.0 or later.  We're too close to 17.0 release.

**Comment by darcywong00** [comment]
For reference, links to Android issues/PRs about the globe experience:
#1887, #5834

Documentation of how the Keyman for Android globe keys behaves:
https://help.keyman.com/products/android/current-version/basic/switching-between-keyboards

**Comment by ermshiperete** [comment]
@mcdurdin Since I won't get to this in the current sprint  I'll move it after the scheduled web-core work.

**Comment by mcdurdin** [comment]
Close as by-design
